tests/src/test/scala/system/utils/KafkaUtils.scala

```diff
@@ -176,7 +189,7 @@ object KafkaUtils {
         val security_protocol = ("security.protocol", "SASL_SSL");
         val keySerializer = ("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
         val valueSerializer = ("value.serializer", "org.apache.kafka.common.serialization.StringSerializer");
-
+        val maxRequestSize = ("max.request.size", "3000000");
```

Comment:
The producer payload size is not big enough to handle the payload in test: `not fire a trigger for a single oversized message`. This went undetected previously as `producer.flush` was not called.